From a7f47ac6ed16e6838edb6f388f47312a905059dd Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Mon, 24 May 2010 11:16:33 +0800
Subject: [PATCH 08/69] ENGR00123740 Exclude the VPU firmware when need codec
 build

When we build codec for rmvb firmware, we should exclude this
common firmware.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 firmware/vpu/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/firmware/vpu/Android.mk b/firmware/vpu/Android.mk
index 5d10d01..592d69d 100644
--- a/firmware/vpu/Android.mk
+++ b/firmware/vpu/Android.mk
@@ -1,5 +1,6 @@
 
 ifeq ($(BOARD_HAVE_VPU),true)
+ifeq ($(EXCLUDED_CODEC_BUILD),false)
 
 LOCAL_PATH := $(call my-dir)
 
@@ -17,3 +18,4 @@ LOCAL_SRC_FILES := $(LOCAL_MODULE)
 include $(BUILD_PREBUILT)
 
 endif
+endif
-- 
1.8.0

